### DISCUSSION TITLE
In the edge edit panel there is a radio button labeled "Controllable"; what is it?

### DISCUSSION THREAD

**Discussion by pacalet** [UPPAALModelChecker/UPPAAL-Meta#120] [Q&A] [answered]
I am using UPPAAL 4.1.20-stratego-9 (rev. 67D95DBCE6B8B4ED), January 2022. In the edge edit panel there is a radio button labeled "Controllable"; what is it? I searched the documentation but could not find anything about this button.

**Comment by RasmusRendal** [comment, marked as answer]
Controllable edges are a [TIGA](https://docs.uppaal.org/extensions/tiga/manual.pdf) feature, which allows the user to simulate timed games.

A simplified version:
![image](https://user-images.githubusercontent.com/19538856/174971965-d57c45a1-8b0a-4cb3-a6b4-b5c04bfc1640.png)

For instance, a timed game might require the player to reach the state Goal, while only "controlling" the controllable edges. Uncontrollable edges are controlled by the environment, and the only way the player can avoid them being taken is by not going into a state where they can be taken

If you do not use any TIGA features, then it is not considered whether an edge is controllable or not.

**Reply by pacalet** [reply]
Thanks a lot. Indeed I never used TIGA but I'll have a look at it.
